### PR TITLE
Initialise Elixir Project

### PR DESCRIPTION
**Pull request by SouzaWander**
This pull request aims to initialise the Elixir project within the repository.

### CODE REVIEW

File: lib/wrapeth/application.ex
Comment (joaop21) [inline lib/wrapeth/application.ex:1]: I don't think we need an application file for now. This is a tool that other projects will use. I believe we won't need to spawn any processes.

File: mix.exs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Wrapeth.Application, []}
```
Comment (joaop21) [inline mix.exs:18]: If you remove the application file, you must also remove this line.

**Comment by joaop21** [comment]
Another thing is that you should add a PR description and a better PR name.

File: lib/wrapeth/application.ex
Comment (SouzaWander) [reply lib/wrapeth/application.ex:1]: Done 🫣

**Review by DavideSilva — APPROVED** [review]
👍

**Pull request merged**